tests/k8s_testcases/example_policies/services1/services1-scheme.yaml

```diff
@@ -0,0 +1,102 @@
+namespaceList: ../../example_podlist/services1/ns_list.yaml
+podList: ../../example_podlist/services1/
+
+networkConfigList:
+  - name: ip
+    networkPolicyList:
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np0
+    networkPolicyList:
+      - services1-networkpolicy0.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np1
+    networkPolicyList:
+      - services1-networkpolicy1.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+  - name: np2
+    networkPolicyList:
+      - services1-networkpolicy2.yaml
+      - policies.yaml
+    expectedWarnings: 0
+
+queries:
+  - name: connectivity_map
+    connectivityMap:
+    - ip
+    - np0
+    - np1
+    - np2
+    expected: 0
+    outputConfiguration:
+      outputFormat: txt
+      fwRulesRunInTestMode: false
+    expectedOutput: ../expected_output/services1_connectivity_map.txt
+
+  - name: sanity
+    sanity:
+      - np1
+    expected: 2
+
+  - name: compare_np1_and_np2
+    twoWayContainment:
+      - np1
+      - np2
+    expected: 1
+
+  - name: np1_and_np2_are_not_strongly_equivalent
+    strongEquivalence:
+      - np1
+      - np2
+    expected: 0
```

Comment:
Why does the expected result say that np1 and np2 are strongly equivalent ?

Reply:
When examining np1 and np2 per policy, they actually were equivalent (since their second policy, allow-webhook-via-ingress, does not allow any connections, since  np1 has empty Egress, and np2 has empty Ingress (mentioned in policyTypes, but not specified). 
I changed the name of second policy in np2 to allow-webhook-via-egress, and now they are not strongly equivalent (since policy names are checked in the query).